cmd/agent/subcommands/run/command.go

```diff
@@ -193,6 +194,7 @@ func run(log log.Component,
 	sysprobeconfig sysprobeconfig.Component,
 	server dogstatsdServer.Component,
 	_ replay.Component,
+	_ pidmap.Component,
```

Comment:
As `newPidMap()` does nothing except returning a type, this line of code is not needed.
Having `_` makes sense when the constructor of the component performs something or when the component have lifecycle hooks.